volatility3/framework/symbols/windows/extensions/registry.py

```diff
@@ -52,6 +77,24 @@ class RegKeyFlags(enum.IntEnum):
     KEY_VIRTUAL_STORE = 0x200
 
 
+class UnexpectedValueTypeException(exceptions.VolatilityException):
```

Comment:
I wondering if this shouldn't live in `exceptions` rather than off on its own here?